packages/frontend/src/components/FlowRow/index.tsx

```diff
@@ -103,7 +103,7 @@ export default function FlowRow(props: FlowRowProps): ReactElement {
               </VStack>
             </GridItem>
             <GridItem area="menu">
-              <Flex alignItems="center" gap={1.5}>
+              <Flex alignItems="center" gap={1.5} justifyContent="flex-end">
```

Comment:
```suggestion
              <Flex alignItems="center" gap={1} justifyContent="flex-end">
```
felt like maybe a bit closer looks nicer but to ur discretion sir